packages/test/test-end-to-end-tests/src/test/directoryEndToEndTests.spec.ts

```diff
@@ -916,7 +917,7 @@ describeNoCompat("SharedDirectory orderSequentially", (getTestObjectProvider) =>
         }
 
         assert.notEqual(error, undefined, "No error");
-        assert.equal(error?.message, errorMessage, "Unexpected error message");
+        assert.equal(error?.message, "Directory with name subDirName already exists", "Unexpected error message");
```

Comment:
@scarlettjlee should this test just be removed in the context of this change? If an op doesn't even get to be created?

Reply:
Yes, remove this test.